### DISCUSSION TITLE
In Android Studio, copilot Chat does not complete the code and stops writing in the middle of writing.

### DISCUSSION THREAD

**Discussion by bluewidy** [community/community#129698] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I was chatting with the co-pilot. I ordered CoPilot to provide me with the entire source code that fixes the problem with my code, but it stops while providing the source code.
<img width="201" alt="화면 캡처 2024-06-25 160333" src="https://github.com/community/community/assets/40682277/fd23a4ae-8508-4991-9877-b2e2bdeff92e">
In fact, if you look to the left of “Generating response”, there is no check mark.

**Comment by Akash1134** [comment]
Hi @bluewidy , 

This sounds like a temporary issue. Restart your IDE and try logging out of Copilot and logging back in. - Sometimes a simple refresh can resolve temporary issues.

Thanks!

**Reply by bluewidy** [reply]
This is not temporary. I've rebooted the computer several times, logged in and logged out, and the same symptoms keep happening. I asked Copilot, and he says he can handle 2048 tokens at once. My code is 171 lines long and has 6487 characters. Is this an amount of length that a Copilot cannot handle?

But the strange thing is, the length of the code that CoPilot outputs is different every time I request it. It's not constant...

https://github.com/community/community/assets/40682277/00a46e05-3c3d-4ef1-b43c-6d5c015ee20b
